consensus/spos/bls/subroundBlock.go

```diff
@@ -107,6 +108,15 @@ func (sr *subroundBlock) doBlockJob() bool {
 	return true
 }
 
+func printLogMessage(ctx context.Context, baseMessage string, err error) {
+	if common.WasContextClosed(ctx) {
+		log.Debug(baseMessage + "context is closing")
```

Comment:
add a space. right now, for this if, it would result in:
DEBUG doBlockJob.createHeadercontext is closing

Reply:
done, thanks